packages/ui/components/ui/tabs.tsx

```diff
@@ -28,7 +28,7 @@ const TabsTrigger = React.forwardRef<
   <TabsPrimitive.Trigger
     ref={ref}
     className={cn(
-      'inline-flex h-9 font-bold text-muted-foreground items-center justify-center whitespace-nowrap rounded-lg ring-offset-background transition-all focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:pointer-events-none disabled:opacity-50 data-[state=active]:bg-teal data-[state=active]:text-muted data-[state=active]:shadow-sm',
+      'inline-flex h-9 font-bold text-muted-foreground items-center justify-center leading-4 rounded-lg ring-offset-background transition-all focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:pointer-events-none disabled:opacity-50 data-[state=active]:bg-teal data-[state=active]:text-muted data-[state=active]:shadow-sm',
```

Comment:
Needed to allow wrapping and tighten up the leading for this view:
![image](https://github.com/penumbra-zone/web/assets/1121544/64a2283c-0cf7-44ce-9f62-c00305bab999)

Reply:
Can we get the scroll bar flush to the right?

Reply:
![image](https://github.com/penumbra-zone/web/assets/1121544/509143fa-c961-4a53-ae6b-d9023c8f80b0)

Fixed. And I also fixed an issue where the bottom buttons overlapped the JSON view.

Reply:
- Can we get bottom margin on the title?
- Why do you think "Reciever's View" does not get proper padding? Can we adjust this a bit?

<img width="366" alt="Screenshot 2024-01-26 at 12 25 37 PM" src="https://github.com/penumbra-zone/web/assets/16624263/aef0797f-5a3d-4408-83e4-18b57c89230f">